src/Microsoft.PowerShell.Commands.Utility/commands/utility/Tee-Object.cs

```diff
@@ -85,6 +86,22 @@ public string Variable
 
         private string _variable;
 
+        /// <summary>
+        /// Sets the encoding to use when writing the file.
+        /// </summary>
+        [Parameter]
+        [ArgumentToEncodingTransformationAttribute]
+        [ArgumentEncodingCompletionsAttribute]
+        [ValidateNotNullOrEmpty]
+        public Encoding Encoding
+        {
+            get { return _encoding; }
+
+            set { _encoding = value; }
+        }
+
+        private Encoding _encoding = ClrFacade.GetDefaultEncoding();
```

Comment:
Please use auto-implemented property:
```suggestion
        public Encoding Encoding { get; set; } = ClrFacade.GetDefaultEncoding();
```

Reply:
This is fixed.